atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraExpiringKeyValueService.java

```diff
@@ -64,7 +64,8 @@ protected CassandraExpiringKeyValueService(
             CassandraKeyValueServiceConfigManager configManager,
             Optional<CassandraJmxCompactionManager> compactionManager,
             Optional<LeaderConfig> leaderConfig) {
-        super(LoggerFactory.getLogger(CassandraKeyValueService.class), configManager, compactionManager, leaderConfig);
+        super(LoggerFactory.getLogger(CassandraExpiringKeyValueService.class),
```

Comment:
Did we mean to change this?  Looks like the logger is passed into the constructor so we can mock it in one test - there's probably a better way to do this.  But in the mean time I think we probably want to keep the logging in the parent class as is.